mount/mount.go

```diff
@@ -75,3 +76,46 @@ func (m *Mount) Mount(target string) error {
 	}
 	return m.mount(target)
 }
+
+// readonlyMounts modifies the received mount options
+// to make them readonly
+func readonlyMounts(mounts []Mount) []Mount {
+	for i, m := range mounts {
+		if m.Type == "overlay" {
+			mounts[i].Options = readonlyOverlay(m.Options)
+			continue
+		}
+		opts := make([]string, 0, len(m.Options))
+		for _, opt := range m.Options {
+			if opt != "rw" {
+				opts = append(opts, opt)
+			}
+		}
+		opts = append(opts, "ro")
```

Comment:
Probably not problematic, but could opts already contain "ro"? (In that case we would have "ro" twice with the current code)

Reply:
good point! I'll change it up and add some test cases.